app/controllers/api/instances_controller.rb

```diff
@@ -0,0 +1,179 @@
+module Api
+  class InstancesController < BaseController
+    def terminate_resource_instances(type, id = nil, _data = nil)
```

Comment:
@abellotti it would be cool if custom actions could be implemented here as `#terminate`, etc... but again, future PR